elements/pfe-card/src/pfe-card.story.js

```diff
@@ -85,7 +85,6 @@ stories.add(PfeCard.tag, () => {
     const ctaPriorityValue = storybookBridge.select(
       "Priority",
       {
-        null: "",
```

Comment:
This is manually building the select field and not using the storybook utils I wrote so if you want it to include the default value, this needs to stay: `null: "default"`